docs/content/en/docs/configuration/command-line.md

```diff
@@ -144,6 +180,13 @@ deprecation. The default behavior is to log all API server warnings.
 
 ---
 
+## --disable-node-list
+
+Disable querying nodes. If `--force-namespace-isolation` is true, this should also be set.  Defaults
```

Comment:
> If `--force-namespace-isolation` is true, this should also be set.

Why is that?

Reply:
You can remove the `--disable-node-list` altogether. Planning a new command-line parsing refactor during v0.14 development which should simplify all this cluster/namespaced scoped configurations, so the description will be improved as well.